task-7/src/server.js

```diff
@@ -0,0 +1,32 @@
+const http = require('http');
+const url = require('url');
+const { metricsController, limitController, defaultController } = require('./controllers');
+
+const limit = 5000;
+
+function startServer(PORT) {
+  const server = http.createServer((req, res) => {
+    res.writeHead(200, { 'Content-Type': 'application/json' });
```

Comment:
```suggestion
  res.setHeader('Content-Type', 'application/json');
```
Вызов метода **writeHead()** записывает все заголовки, после этого невозможно добавление статус кода или других заголовков.
Поэтому лучше просто установить заголовок через **setHeader()**, а записать его и устаноивь статус код  позже, но перед тем как начнется запись тела (body)